dk62_serhiienko/lab2_simplest_kernel_modules/src/jiffies_test.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import time
+import subprocess
+import argparse
+import re
+
+
+def get_logs():
+    kern_log = subprocess.Popen(['cat', '/var/log/kern.log'], stdout=subprocess.PIPE)
```

Comment:
Создавать процесс тут не нужно.
```Python3
# Assume file is short to fit in memory. Otherwise need to seek from the end
with open('/var/log/kern.log') as f:
    return ''.join(list(f)[-3:])
```

Reply:
Грязный хак для sudo: https://stackoverflow.com/a/56362166/5750172